app/src/main/res/layout/item_country_report.xml

```diff
@@ -25,25 +40,37 @@
             android:gravity="center"
             android:paddingBottom="10dp"
             android:text="@{item.country}"
-            android:textSize="20dp"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
+            android:textSize="20sp"
+            app:layout_constraintEnd_toEndOf="@+id/end_guideline"
+            app:layout_constraintStart_toStartOf="@+id/start_guideline"
             app:layout_constraintTop_toTopOf="parent"
             tools:text="China" />
 
+        <ImageView
+            android:id="@+id/bookmark_button"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:src="@drawable/ic_flag"
+            android:onClick="@{() -> item.followed ? listener.onCountryUnfollowed(item.country) : listener.onCountryFollowed(item.country) }"
```

Comment:
This will crash if listener won't be set.

Reply:
It won't. Data binding is smart enough! :)